index.js

```diff
@@ -1,17 +1,66 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    function compare(a, b) {
+      return a - b;
+    }
+    this.items.sort(compare);
+    // Este tambien nos valdría
+    // this.min(this.items);
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (this.items[pos]) {
+      return this.items[pos];
+    } else {
+      throw new Error("OutOfBounds");
+    }
+  }
 
-  max() {}
+  max() {
+    function compare(a, b) {
+      return b - a;
+    }
+    if (this.items.length > 0) {
+      this.items.sort(compare);
+      return this.items[0];
+    } else {
+      throw new Error("EmptySortedList");
+    }
+  }
 
-  min() {}
+  min() {
+    function compare(a, b) {
+      return a - b;
+    }
+    if (this.items.length > 0) {
+      this.items.sort(compare);
+      return this.items[0];
+    } else {
+      throw new Error("EmptySortedList");
+    }
+  }
 
-  sum() {}
+  sum() {
+    const sum = this.items.reduce((ac, valueAc) => {
+      return ac + valueAc;
+    }, 0);
+    return sum;
```

Comment:
Esto es lo mismo que hacer esto
```suggestion
    return this.items.reduce((ac, valueAc) => {
      return ac + valueAc;
    }, 0);
```